utils/run_streaming.sh

```diff
@@ -0,0 +1,2 @@
+#!/bin/bash
```

Comment:
not sure if this file belongs in this PR

Reply:
Let's remove this from this PR for now and add it back once the docker related changes are in.